app/models/avalon_jobs.rb

```diff
@@ -49,10 +49,16 @@ def self.delete_masterfile pid
       logger.warn "MasterFile #{oldpath} does not exist"
     end
   end
+
+  def self.collection_batch_ingest(collection_id)
+    collection = Admin::Collection.find(collection_id)
+    Avalon::Batch::Ingest.new(collection).ingest
+  end
+
 class << self
```

Comment:
This is silly on avalon's part. Every method in the class is a class level method. Why only put some in a `class << self` and others use `self.method`? Stick with one and follow it....

Considering avalon master has tossed avalon_jobs model. Could refactor this?